regression/memory-analyzer/Makefile

```diff
@@ -4,6 +4,7 @@ GOTO_GCC_EXE=../../../src/goto-cc/goto-gcc
 MEMORY_ANALYZER_EXE=../../../src/memory-analyzer/memory-analyzer
 
 clean:
+	find -name '*.gb' -execdir $(RM) '{}' \;
```

Comment:
I had a sneaky feeling that there were a couple of '*.gb' files which were actually test inputs and hence needed to be kept - I can't remember which directory those are in though, so I'm assuming they are not in the memory-analyzer directory? :-)

Reply:
I had the feeling too -- though it turned out to be unfounded :)